src/scriptdev2/scripts/eastern_kingdoms/molten_core/boss_garr.cpp

```diff
@@ -16,8 +16,8 @@
 
 /* ScriptData
 SDName: Boss_Garr
-SD%Complete: 80
-SDComment: Firesworn erruption needs to be revisited
+SD%Complete: 95
+SDComment: 'Summon Player' missing
```

Comment:
Was there a summon player back in Classic for Garr? I know it was only introduced in early TBC for Emerald Dragons and Onyxia. (and in fact, I did not even know he had a summon ability)